### PR TITLE
docs: fix and align section names

### PR DESCRIPTION
**Pull request by duesee**
**Overall Review of Changes:**
Minor improvements to casing, and a fix for a section title.

**Issue Fixes:**
None, (should I open first?).

**Enhancements:**
Playbook is now better aligned to the document.

**How has this been tested?:**
I didn't test because I believe no tests are required.

### CODE REVIEW

File: tasks/section_1/main.yml [hunk 1]
```diff
@@ -5,46 +5,46 @@
...
   when: not system_is_container
 
-- name: "SECTION | 1.1.8 | configure /dev/shm"
+- name: "SECTION | 1.1.8 | Configure /dev/shm"
   ansible.builtin.import_tasks:
       file: cis_1.1.8.x.yml
   when: not system_is_container
 
-- name: "SECTION | 1.1.9 | configure software updates"
+- name: "SECTION | 1.1.9 | Disable Automounting"
```
Comment (duesee) [inline tasks/section_1/main.yml:42]: I believe this slipped through. This is the only major change here.

File: tasks/section_1/main.yml [hunk 2]
```diff
@@ -5,46 +5,46 @@
...
   ansible.builtin.import_tasks:
       file: cis_1.1.9.yml
   when: not system_is_container
 
-- name: "SECTION | 1.1.10 | Disable USB storage"
+- name: "SECTION | 1.1.10 | Disable USB Storage"
```
Comment (duesee) [inline tasks/section_1/main.yml:47]: `cis_1.1.10.yml` has the same capitalization.

**Review by uk-bolly — APPROVED** [review]
Nice tidy up thank you

**Comment by duesee** [comment]
> hi @duesee
> 
> Thank you so much for taking the time for this PR and nice nice spots. Unfortunately the commits are not GPG signed which are required in order to merge this PR. If you are able to resolve this (it will show verified next to each commit in the list).
> 
> Many thanks
> 
> uk-bolly

Just configured commit signing. Thanks for the patience!

**Review by uk-bolly — APPROVED** [review]
thank you

**Pull request merged**